### PR TITLE
added tiny llama examples for lora and qlora

### PR DESCRIPTION
**Pull request by tdolan21**
example for TinyLlama1.1B using lora and qlora. Completes in roughly 10 minutes on a 4090.

### CODE REVIEW

File: examples/tiny-llama/qlora.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+saves_per_epoch: 1
+debug:
+deepspeed:
+weight_decay: 0.0
+fsdp:
+fsdp_config:
+special_tokens:
+  bos_token: "<s>"
+  eos_token: "</s>"
+  unk_token: "<unk>"
```
Comment (winglian) [inline examples/tiny-llama/qlora.yml]:
```suggestion
special_tokens:
```
these are un-necessary

File: examples/tiny-llama/lora.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+saves_per_epoch: 1
+debug:
+deepspeed:
+weight_decay: 0.0
+fsdp:
+fsdp_config:
+special_tokens:
+  bos_token: "<s>"
+  eos_token: "</s>"
+  unk_token: "<unk>"
```
Comment (winglian) [inline examples/tiny-llama/lora.yml]:
```suggestion
special_tokens:
```

**Comment by winglian** [comment]
can you also remove the existing tiny-llama yml here too pls? https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/examples/llama-2/tiny-llama.yml

**Review by winglian — APPROVED** [review]
Thank you!

**Comment by ehartford** [comment]
Awesome!

**Pull request merged**